src/Sulu/Bundle/AdminBundle/Metadata/SchemaMetadata/ArrayMetadata.php

```diff
@@ -33,12 +38,14 @@ public function __construct(
         bool $mandatory,
         SchemaMetadata $schemaMetadata,
         int $minItems = null,
+        int $maxItems = null,
         bool $uniqueItems = null
```

Comment:
Since the parameters `$minItems` and `$uniqueItems` are not released yet, I decided to add `$maxItems` in the middle of them instead of at the end